test/suite/test_assert06.py

```diff
@@ -33,15 +33,18 @@
 from suite_subprocess import suite_subprocess
 import wiredtiger, wttest
 from wtscenario import make_scenarios
+from wtdataset import SimpleDataSet
```

Comment:
It seems strange to leave this one test in the `test_assert06.py` when all the rest were deleted and moved to the other test. There is a `test_assert07.py` that seems mis-categorized. Consider renaming these two.

Reply:
Honestly, I ran out of steam after rewriting all of the others. It was pretty painful. :(